pkg/messageFormat/ast.go

```diff
@@ -0,0 +1,101 @@
+package messageFormat
```

Comment:
Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.